spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-fabric8-client-configmap/src/main/resources/application.yaml

```diff
@@ -0,0 +1,5 @@
+spring:
```

Comment:
this test was reading from a configmap and asserting the value there. so why not move this one to config data? it would be the first integration test we have that has moved to config data